cmd/istio-cni/main.go

```diff
@@ -55,6 +55,7 @@ type Kubernetes struct {
 	NodeName             string   `json:"node_name"`
 	ExcludeNamespaces    []string `json:"exclude_namespaces"`
 	CniBinDir            string   `json:"cni_bin_dir"`
+	ExcludeIPRanges      string   `json:"exclude_ip_ranges"`
```

Comment:
This option has nothing to do with this structure, which is specific to the Kubernetes environment.